### PR TITLE
Deployment scripts

### PR DESCRIPTION
**Pull request by Paul4912**
# Motivation # 
Deploy scripts have not been updated since pools v1. We need to update them so they work with pools v2. 

# Scope #
This PR will just modify deploy script. When i try deploying on testnet i might have errors i need to fix will probably put in another pr if this gets merged early.

# Changes #
- Update parameters so deploy scripts are up to date with contract changes
- Add invariant check into deploy script
- Change `deployPool` event to include `poolCommitter`
- Deploy SMA oracle

### CODE REVIEW

File: deploy/ArbitrumDeploy.js [hunk 1]
```diff
@@ -55,9 +55,42 @@ module.exports = async (hre) => {
...
+        contract: "PriceObserver",
+    })
+
+    // deploy SMA Oracle
+    const smaOracleWrapper = await deploy("EthUsdPriceSMAOracle", {
+        from: deployer,
+        log: true,
+        contract: "SMAOracle",
+        args: [
+            RinkebyEthUsdOracle.address,
```
Comment (sporejack) [inline deploy/ArbitrumDeploy.js:74]: The network here differs from the one on line 58 (i.e., Rinkeby versus Kovan, respectively). Is this intentional?

File: deploy/ArbitrumDeploy.js [hunk 2]
```diff
@@ -55,9 +55,42 @@ module.exports = async (hre) => {
...
+            8,
+            priceObserver.address,
+            12,
+            3600,
```
Comment (sporejack) [inline deploy/ArbitrumDeploy.js:78]: Inline comments next to these parameters would be greatly beneficial to the readability of this script.

**Review by sporejack — CHANGES_REQUESTED** [review]

File: deploy/ArbitrumDeploy.js (hunk 1 above)
Comment (Paul4912) [reply deploy/ArbitrumDeploy.js:74]: a little, i was mainly focused on making the logic correct and then maybe make another pr after i deploy to use correct constants for addresses. to my knowledge we deploying on arbitrum rinkeby.

Comment (Paul4912) [reply deploy/ArbitrumDeploy.js]: I looked at the rinkeby oracles and they are eth rinkeby not arbitrum rinkeby. Will need to find the rinkeby oracles on aribitrum.

**Review by mynameuhh — APPROVED** [review]

File: deploy/ArbitrumDeploy.js (hunk 1 above)
Comment (Paul4912) [reply deploy/ArbitrumDeploy.js]: alright updated with arbitrum rinkeby oracle addresses

**Review by sporejack — APPROVED** [review]

**Pull request merged**